### ISSUE TITLE
This app is no longer being supported

### ISSUE THREAD

**Issue by jnaviamtc** [molenzwiebel/Mimic#54]
As the title states. Love this app, still use it, but it's so abandoned consider it a miracle it still works semi-properly.

If you're not down to constantly refresh so it works, move along.

**Comment by Soulivan71** [comment]
Wich one ?

**Comment by harmonherring** [comment]
Are there any other apps that do this?

**Comment by espentruls** [comment]
Yes, overwolf has taken over and made an updated version over at their app.

**Comment by Chosey98** [comment]
@espentruls Can you send the link please

**Comment by espentruls** [comment]
https://www.overwolf.com/app/Mao-Tou_Projects_Ltd-Mimic
